src/main/java/com/xeeva/catalog/RequisitioningPage.java

```
@@ -1,763 +1,764 @@
+			}
+		}
+
+		return returnValue;
+	}
+
+	/**
+	 * @summary: Method to click on Show Favourite items.
+	 * @author: Praveen Namburi, @version: Created 09-09-2016
+	 */
+	public void clickShowFavouriteItems(){
+		lnkShowFavItems.syncVisible(15, false);
+		lnkShowFavItems.click();
+
+	}
+
+	/**
+	 * @summary: Method to add price agreement from favourite folder.
+	 * @author: Praveen Namburi,@version: Created 09-09-2016
+	 * @param strUOMValue
+	 */
+	public void addPriceAgreementItemsFromFavFolder(String strUOMValue){
+		pageLoaded();
+		String getLocalItemsCount = lblLocalItems.getText();
+		TestReporter.log("Local-Items Count is: "+ getLocalItemsCount);
+
+		String getGlobalItemsCount = lblGlobalItems.getText();
+		TestReporter.log("Global-Items Count is: "+ getGlobalItemsCount);
+
+		if(!getLocalItemsCount.contains("0")){
+			TestReporter.log("Click on Local-Items tab.");
+			LocalItemsTab localItem = new LocalItemsTab(driver);
+			//localItem.click_localItemsTab();
+
+			String itemNumber = null;
+			List<WebElement> localItems = localItemsGrid;
+			for(WebElement inputItem : localItems){
+				itemNumber = inputItem.getText();
+				TestReporter.log("Item number is: "+itemNumber);
+				//Select the UOM option from the listbox.
+				List<WebElement> listboxUOMs = driver.findElements(By.xpath("//select[@class='textFieldList width90Px']"));
+				Select select = new Select(listboxUOMs.get(0));
+				select.selectByVisibleText(strUOMValue);
+				//new Select(listboxUOMs.get(0)).selectByVisibleText(strUOMValue);
+				//Click on Add-to-cart link.
+				List<WebElement> localAddToCartItemLinks = lstAddToCartItemsGrid;
+				if(localAddToCartItemLinks.size()>0){
+					for(WebElement linkAddToCartItem :localAddToCartItemLinks){
+						TestReporter.log("Click on 'Add-To-Cart' link.");
+						linkAddToCartItem.click();
+						break;
+					}
+				}
+				break;
+			}
+		}else if(getLocalItemsCount.contains("0")){
+			TestReporter.log(" 'No Records Found !!' in Local Items tab. "
```

Does this need to be a test failure instead of just reporting a log statement?